extras/pcp/foreman-hotproc.conf

```diff
@@ -0,0 +1,14 @@
+#pmdahotproc
+Version 1.0
+
+(
+  fname == "java" ||
+  fname == "redis-server" ||
+  (fname == "postgres" && psargs ~ /-D/) ||
+  psargs ~ /puma/ ||
+  psargs ~ /pulpcore.app.wsgi:application/ ||
+  psargs ~ /pulpcore.content:server/ ||
+  psargs ~ /pulpcore-worker/ ||
```

Comment:
yes, this includes pulpcore which is Katello only, but it doesn't hurt during collection and PCP only supports having one hotproc config

Reply:
I'm fine with that. This could get needlessly complex trying to account for every plugin in it's own repo by the sounds of it.